Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.5
+ENV PYTHONUNBUFFERED 1
+RUN mkdir /code
+WORKDIR /code
+ADD requirements.txt /code/
+RUN pip install -r requirements.txt
```

Comment:
Hey @joaoricardo000, you mean letting the CMD (`python manage.py runserver` etc) on the Dockerfile?